lib/ansible/modules/cloud/azure/azure_rm_securitygroup.py

```diff
@@ -73,10 +73,14 @@
             source_port_range:
                 description:
                   - Port or range of ports from which traffic originates.
+                aliases:
+                  - source_port_ranges
                 default: "*"
```

Comment:
why to have alias? name could just stay the same and description change

Reply:
Yes you are right, I will remove it